CHANGELOG.md

```diff
@@ -2,36 +2,79 @@
+- Fix loading app with autocomplete disabled ([#1650](https://github.com/thelounge/lounge/pull/1650) by [@dgw](https://github.com/dgw))
+- Fix URL query parameters in public mode ([#1661](https://github.com/thelounge/lounge/pull/1661) by [@MaxLeiter](https://github.com/MaxLeiter))
+- Fix hyphenated names overflowing (#1667) ([#1671](https://github.com/thelounge/lounge/pull/1671) by [@LFlare](https://github.com/LFlare))
+- Fix missing attributes on unhandled messages ([#1695](https://github.com/thelounge/lounge/pull/1695) by [@xPaw](https://github.com/xPaw))
+- Correctly display kicks when kicker is server ([#1693](https://github.com/thelounge/lounge/pull/1693) by [@xPaw](https://github.com/xPaw))
+- Go back to writing user files synchronously ([#1701](https://github.com/thelounge/lounge/pull/1701) by [@xPaw](https://github.com/xPaw))
+- Fix local theme folder ([#1706](https://github.com/thelounge/lounge/pull/1706) by [@xPaw](https://github.com/xPaw))
+
 ### Documentation
+
 In the main repository:
-On the website:
-### Internals
-```
 
-All sections are explained on the link above, they are all optional, and each of them should contain a list of PRs formatted as such:
+- Fix "help wanted" link in CONTRIBUTING file ([#1673](https://github.com/thelounge/lounge/pull/1673) by [@timmw](https://github.com/timmw))
+- Document autocompletion in the help page ([#1609](https://github.com/thelounge/lounge/pull/1609) by [@dgw](https://github.com/dgw))
+- Add a script to pre-generate changelog entries ([#1707](https://github.com/thelounge/lounge/pull/1707) by [@astorije](https://github.com/astorije))
 
-```md
-- Description ([#PR_NUMBER](https://github.com/thelounge/lounge/pull/PR_NUMBER) by [@GITHUB_USERNAME](https://github.com/GITHUB_USERNAME))
-```
--->
+### Internals
 
-<!-- New entries go after this line -->
+- Remove channel containers from DOM after quitting network ([#1607](https://github.com/thelounge/lounge/pull/1607) by [@PolarizedIons](https://github.com/PolarizedIons))
+- Create public folder with Webpack ([#1611](https://github.com/thelounge/lounge/pull/1611) by [@xPaw](https://github.com/xPaw))
+- Cleanup client manager functions ([#1636](https://github.com/thelounge/lounge/pull/1636) by [@xPaw](https://github.com/xPaw))
+- Add Node.js v9 testing to Travis builds ([#1678](https://github.com/thelounge/lounge/pull/1678) by [@astorije](https://github.com/astorije))
+- Remove `webpack-cleanup-plugin` ([#1682](https://github.com/thelounge/lounge/pull/1682) by [@xPaw](https://github.com/xPaw))
+- Keep `findLinks.js` file in npm builds ([#1704](https://github.com/thelounge/lounge/pull/1704) by [@xPaw](https://github.com/xPaw), [#1705](https://github.com/thelounge/lounge/pull/1705) by [@astorije](https://github.com/astorije))
+- Allow `run-pr` script to pass arguments to `npm start` ([#1662](https://github.com/thelounge/lounge/pull/1662) by [@MaxLeiter](https://github.com/MaxLeiter))
```

Comment:
Do we really need this in the changelog?

Reply:
Yeah. I don't like to hide things from the changelog (if reader wants a curated version, introduction is meant for that). Plus it's in the "Internals" section, which we added at the bottom to avoid clogging other sections with this kind of things, so that's fine.